process/errors.go

```diff
@@ -292,8 +292,11 @@ var ErrNilPrevRandSeed = errors.New("provided previous rand seed is nil")
 // ErrNilRequestHeaderHandlerByNonce signals that a nil header request handler by nonce func was provided
 var ErrNilRequestHeaderHandlerByNonce = errors.New("nil request header handler by nonce")
 
-// ErrLowShardHeaderRound signals that shard header round is too low for processing
-var ErrLowShardHeaderRound = errors.New("shard header round is lower than last committed for this shard")
+// ErrLowerRoundInNotarizedBlock signals that a notarized header round is too low for processing
```

Comment:
why notarized? the block is not already notarized by metachain

Reply:
This error is used when you check if the blocks from the "other" chain (if you are a shard node and you check if the meta blocks synced are valid or if you are a meta node and you check if the shard blocks synced are also valid). This is because we are using one method for both chains as the code is common and this is why this method is placed in baseProcess class. On the other hand, when a node checks if its chain blocks are valid, it will use the below error: ErrLowerRoundInBlock, to easily differentiate the location of the problem when somebody tries to find a problem(debugging) using the log files or terminal.

Reply:
Then we should use some other naming, notarization for me is done on Metachain.
E.g ErrLowerRoundInOtherChainBlock

Reply:
Done